redisai/commands_test.go

```diff
@@ -1180,7 +1180,8 @@ func TestCommand_DagRun(t *testing.T) {
 		{"t_load", args{[]string{"persisted_tensor_1"}, []string{"tensor1"}, NewDag().TensorSet("tensor1", TypeFloat32, []int64{1, 2}, []int64{5, 10})}, false},
 		{"t_load_err", args{[]string{"not_exits_tensor"}, []string{"tensor1"}, NewDag().TensorSet("tensor1", TypeFloat32, []int64{1, 2}, []int64{5, 10})}, true},
 		{"t1", args{nil, nil, NewDag().TensorSet("a", TypeFloat32, []int64{1}, []float32{1.1})}, false},
-		{"t_blob", args{nil, nil, NewDag().TensorSet("a", TypeFloat32, []int64{1}, []float32{1.1}).TensorSet("b", TypeFloat32, []int64{1}, []float32{4.4}).ModelRun("test:DagRun:mymodel:1", []string{"a", "b"}, []string{"mul"}).TensorGet("mul", TensorContentTypeBlob)}, false},
+		{"t_blob_run", args{nil, nil, NewDag().TensorSet("a", TypeFloat32, []int64{1}, []float32{1.1}).TensorSet("b", TypeFloat32, []int64{1}, []float32{4.4}).ModelRun("test:DagRun:mymodel:1", []string{"a", "b"}, []string{"mul"}).TensorGet("mul", TensorContentTypeBlob)}, false},
+		{"t_blob_execute", args{nil, nil, NewDag().TensorSet("a", TypeFloat32, []int64{1}, []float32{1.1}).TensorSet("b", TypeFloat32, []int64{1}, []float32{4.4}).ModelExecute("test:DagRun:mymodel:1", []string{"a", "b"}, []string{"mul"}, 0).TensorGet("mul", TensorContentTypeBlob)}, true},
```

Comment:
Why do you use here `model_execute` inside deprecated `dag_run`? How do you verify that the deprecated `scriptrun` and `modelrun` commands are not a part of `DAGexecute` (and vice versa)?

Reply:
I use `model_execute` inside deprecated `dag_run` to test that  `dag_run`  is failing when using `model_execute`. 
I can't verify it during the command's building, but I tested that an error occurs when combining `model_execute` with `dag_run` or `model_run` with `dag_execute.`

We don't have support to `script_run` in dag, so it is easier in this case (we didn't add it to the python client also...)